WORKSPACE

```diff
@@ -106,3 +106,13 @@ new_http_archive(
     type = "zip",
     url = "https://github.com/nunit/nunitv2/releases/download/2.6.4/NUnit-2.6.4.zip",
 )
+
+maven_jar(
+    name = "scala-xml",
+    artifact = "org.scala-lang.modules:scala-xml_2.11:1.0.5",
```

Comment:
Oh, nice! Specifying the scala version to use in inside the workspace rather than a symlink sounds great.

+1 regarding the plan once both are merged.